platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -123,6 +123,9 @@ typedef struct odp_packet_hdr_t {
 	/* Flow hash value */
 	uint32_t flow_hash;
 
+	/* Classifier mark */
+	uint16_t cls_mark;
```

Comment:
Initialization to zero is missing from packet_init().

Reply:
Done in v2. Uses a flag for maintaining packet input / alloc performance.